metadata_service/proxy/atlas_proxy.py

```diff
@@ -416,9 +459,41 @@ def get_tags(self) -> List:
                 )
         return tags
 
-    def get_table_by_user_relation(self, *, user_email: str,
-                                   relation_type: UserResourceRel) -> Dict[str, Any]:
-        pass
+    def get_table_by_user_relation(self, *, user_email: str, relation_type: UserResourceRel) -> Dict[str, Any]:
+        params = {
+            'typeName': 'Reader',
+            'offset': '0',
+            'limit': '50',
+            'entityFilters': {
+                'condition': 'AND',
+                'criterion': [
+                    {
+                        'attributeName': 'qualifiedName',
+                        'operator': 'contains',
+                        'attributeValue': user_email
+                    },
+                    {
+                        'attributeName': 'isFollowing',
+                        'operator': 'eq',
+                        'attributeValue': 'true'
+                    }
+                ]
+            },
+            'attributes': ['count', 'qualifiedName']
+        }
+
+        search_results = self._driver.search_basic.create(data=params)
```

Comment:
Can you please put a comment that `.create()` does not actually create entities, but a post method for basic search. It took me some time to realize that. (although I wrote this myself in the first place :D )